app/models/auth_provider.rb

```diff
@@ -29,6 +29,8 @@
 class AuthProvider < ApplicationRecord
   belongs_to :creator, class_name: "User"
 
+  has_many :user_auth_provider_links, dependent: :destroy
```

Comment:
:green_circle: Because that's possible now:

```suggestion
  has_many :user_auth_provider_links, dependent: :destroy
  has_many :users, through: :user_auth_provider_links
```